inst/examples/circosVcfData/app.R

```diff
@@ -0,0 +1,164 @@
+suppressPackageStartupMessages(library(StructuralVariantAnnotation))
+suppressPackageStartupMessages(require(ggbio))
+library(shiny)
+library(shiny.gosling)
+use_gosling(clear_files = FALSE)
+
+colo829_vcf <- VariantAnnotation::readVcf(
```

Comment:
I suppose that `colo829` is a cell line?
WDYT if we add a comment with some context? 
```suggestion
# Reading data from cell line COLO829. See more info about this cell line at https://www.cellosaurus.org/CVCL_1137
colo829_vcf <- VariantAnnotation::readVcf(
```